docs/webmaster-modules.md

```diff
@@ -0,0 +1,107 @@
+# Webmaster modules
+
+Administrators on webmaster libraries has the ability to upload and
+enable extra modules. These can be both standard Drupal contrib
+modules, or modules developed by/for the library.
+
+Developing bespoke modules doesn't not differ from developing modules
+in general, as installing and configuring modules works as expected.
+
+However, it not uncommon for the developer to wish to provide some
+configuration "outside" the modules own configuration, adding taxonomy
+vocabularies, adding fields to nodes or even changing basic
+configuration variables, which is somewhat challenging as DPL controls
+the configuration.
+
+So to avoid every module doing it differently, this guide exists. And
+if enough modules finds the need, the technique described here could
+be implemented in a `dpl_webmaster` module that provide it as a
+service.
```

Comment:
I had hoped that the code needed for configuration handling would be so simple that we could include it in the documentation.

I see that is not the case so referencing the Brug Byen module makes sense. It does point at infrastructure which could be provided by a central solution going forward.